File: BasicAndroidArchitecture/myfeaturemodule/src/main/java/com/example/myfeaturemodule/models/DummyResponse.java
package com.example.myfeaturemodule.models;

import com.google.gson.annotations.SerializedName;

public class DummyResponse {

    @SerializedName("id")
    private long id;

    @SerializedName("employee_name")
    private String name;

    @SerializedName("employee_salary")
    private int salary;

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public int getSalary() {
        return salary;
    }
}

File: RepositoryPattern/SampleApplicationForGM/myfeature/src/main/java/com/example/myfeature/models/DummyResponse.kt
package com.example.myfeature.models

import com.google.gson.annotations.SerializedName

class DummyResponse {
    @SerializedName("id")
    val id: Long = 0

    @SerializedName("employee_name")
    val name: String? = null

    @SerializedName("employee_salary")
    val salary = 0

}